### PR TITLE
[CSLD-41] Fix PoS and Minting description

### PR DESCRIPTION
**Pull request by ChShersh**

### CODE REVIEW

File: _docs/2017-01-01-index.md [hunk 1]
```diff
@@ -151,9 +151,11 @@ which is a worthy read for anyone with a background in cryptocurrency
 theory. The core idea of Proof of Stake is that instead of wasting
 electricity on cracking computationally heavy problems, a node is
 selected to mint a new block, with a probability proportional
-to the amount of coins a particular node has. A node that attempts to
-generate a block is called _a slotleader_. You can read more about
-this process in [Proof of Stake in Cardano SL](/cardano/proof-of-stake/).
+to the amount of coins a particular node has. Every node of network can attempt to
```
Comment (gromakovsky) [inline _docs/2017-01-01-index.md]: Well, node can make an attempt, but there is no reason to do it unless you are a slot leader :/

File: _docs/2017-01-01-index.md [hunk 2]
```diff
@@ -151,9 +151,11 @@ which is a worthy read for anyone with a background in cryptocurrency
...
+generate a block. If node has positive [stake](/cardano/proof-of-stake/#stake) it
+is called _a stakeholder_. If node eventually becomes chosen to mint block
+it becomes _slotleader_. You can read more about this process in
```
Comment (gromakovsky) [inline _docs/2017-01-01-index.md]: Well, being chosen to mint block and being a slotleader (btw, is it intentional that there is no space?) are the same things actually. This sentence is like `if you become chosen to rule a country you become a president`. I am not sure about usefulness of such sentence.

File: _docs/2017-01-01-index.md [hunk 3]
```diff
@@ -151,10 +151,10 @@ which is a worthy read for anyone with a background in cryptocurrency
...
 electricity on cracking computationally heavy problems, a node is
 selected to mint a new block, with a probability proportional
-to the amount of coins a particular node has. Every node of network can attempt to
-generate a block. If node has positive [stake](/cardano/proof-of-stake/#stake) it
-is called _a stakeholder_. If node eventually becomes chosen to mint block
-it becomes _slotleader_. You can read more about this process in
+to the amount of coins a particular node has. If node has positive
+[stake](/cardano/proof-of-stake/#stake) it is called _a stakeholder_.
+If node eventually becomes chosen to mint block
+it is called _a slotleader_. You can read more about this process in
```
Comment (ChShersh) [inline _docs/2017-01-01-index.md]: @gromakovsky I rewrote sentences in a different way. I didn't see definitions of _stakeholder_ and _slotleader_ earlier here, seems good place to introduce them. Also, we can't assume that people read paper before reading docs.

File: _docs/2017-01-01-index.md [hunk 4]
```diff
@@ -151,9 +151,11 @@ which is a worthy read for anyone with a background in cryptocurrency
 theory. The core idea of Proof of Stake is that instead of wasting
 electricity on cracking computationally heavy problems, a node is
 selected to mint a new block, with a probability proportional
-to the amount of coins a particular node has. A node that attempts to
-generate a block is called _a slotleader_. You can read more about
-this process in [Proof of Stake in Cardano SL](/cardano/proof-of-stake/).
+to the amount of coins a particular node has. Every node of network can attempt to
+generate a block. If node has positive [stake](/cardano/proof-of-stake/#stake) it
```
Comment (ChShersh) [inline _docs/2017-01-01-index.md]: @rockbmb I just copied this definition from @gromakovsky from here https://serokell.slack.com/archives/hackathon-docs/p1486848893000318
> Stakeholder is someone who has positive stake

Positive means `> 0`

Comment (rockbmb) [inline _docs/2017-01-01-index.md]: @ChShersh Ah, understood. Can you add that inside parens after the word?

Comment (ChShersh) [inline _docs/2017-01-01-index.md]: @rockbmb Fixed. But I don't know how to write it nicer so I guess it'll be enough.

**Pull request merged**